i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/deployment/README.md

```diff
@@ -2,8 +2,34 @@
 sidebar_position: 8
 ---
 
-# 🚀 部署 Logto
+# 🚀 部署
 
-:::info
-本文档仍在撰写中。
-:::
+## 环境变量
+
+我们在演示中（`docker-compose.yml`）使用了一组自动生成的环境变量，你应该替换成自己的，并维护多个 Logto 实例之间的变量统一性。
+
+你可以直接设置环境变量，或者在 Logto 项目根目录中放置一个 `.env` 文件。如果你在用 Docker 进行测试，可以去 `/etc/logto` 里看看自动生成的 `.env`。
+
+### 基础
+
+- `DB_URL` Logto 数据库的 [Postgres DSN](https://www.postgresql.org/docs/14/libpq-connect.html#id-1.7.3.8.3.6)。
+- `PORT` Logto 监听的本地端口。默认 `3001`。
+- `ENDPOINT` 你可以指定一个带有自定义域名的指向 Logto 的 URL，用于在线测试或生产环境。这也会影响到 [OIDC issuer identifier](https://openid.net/specs/openid-connect-core-1_0.html#IssuerIdentifier) 和「管理控制台」Redirect URIs 的值。
```

Comment:
Maybe more specific? Tell the user they should specify the host domain to this variable. e.g. `ENDPOINT=<https://your-domain.com>`

Reply:
yes